package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Memory leak testing for node. Javascript memory footprinting using your favorite test runner.",
   "main": "lib/index",
   "scripts": {
-    "test": "mocha --timeout 10000 test/ && standard src/**/*.js test/**/*.js"
+    "test": "mocha --timeout 60000 test/ && standard src/**/*.js test/**/*.js"
```

Comment:
Did you run into timeouts?

Reply:
yes. indeed, even with my latest Macbook Pro... It took me around ~10s to run the test, so I do have experienced timeout several times. So for slower machines, I make it to 60s.